README.md

```diff
@@ -31,6 +31,11 @@ Also see [Cargo.toml](Cargo.toml)
 ```
 ~$ cargo install --path=. --force
 ```
+#### For single user with cargo
+
+```
+~$ cargo install --git https://github.com/danielronalds/joshuto.git --force
```

Comment:
Link should probably be https://github.com/kamiyaa/joshuto.git

But other than that, lgtm